src/Sulu/Bundle/ResourceBundle/Tests/Functional/Controller/OperatorControllerTest.php

```diff
@@ -133,7 +133,7 @@ public function testCget()
     {
         $this->client->request(
             'GET',
-            '/api/operators'
+            '/api/operators?locale=de'
```

Comment:
Like with the FilterController, the request does not fail, when no locale is provided.